### ISSUE TITLE
comparison between vecty and vugu?

### ISSUE THREAD

**Issue by xoviat** [hexops/vecty#257]
I had planned to eventually write google material components for vecty. However, there is a new project called vugu that seems to be progressing much more rapidly than vecty: http://github.com/vugu/vugu. Perhaps with this new project, it is not such a good idea to bet on vecty anymore.

**Comment by nobonobo** [comment]
I love vecty! Because, I can use the Go tools for vecty and vecty components.

**Comment by xoviat** [comment]
Yeah, the problem with vugu is that it has a new html/go syntax that's unsupported by the editors. But it also has a fancy website (not written in vugu) that makes it seem like a more mature project.

**Comment by slimsag** [comment]
Will indeed add a comparison page once we have the new website up.

**Comment by hirohido** [comment]
> Yeah, the problem with vugu is that it has a new html/go syntax that's unsupported by the editors. But it also has a fancy website (not written in vugu) that makes it seem like a more mature project.

Honestly thought the vugu website was made with vugu  💔
